wiki/Tournaments/oPHF/2018/en.md

```diff
@@ -1281,11 +1281,11 @@ Detailed results for this round are found via [this link](https://docs.google.co
 #### Match format
 
 - Players will alternate picks until a match winner is determined. This will be in a Best-of-X series with the formats listed as follows:
-  - Match week 1 - Best-of-9 (First to 5)
-  - Match week 2 - Best-of-11 (First to 6)
-  - Match week 3 - Best-of-11 (First to 6)
-  - Match week 4 - Best-of-13 (First to 7)
-  - Match week 5 - Best-of-13 (First to 7)
+  - Match week 1—Best-of-9 (First to 5)
```

Comment:
A table might look best here.

Reply:
there's more to clean up in the article - either use colons or revert for now

Reply:
```suggestion
  - Match week 1: Best-of-9 (First to 5)
```

Reply:
I wasn't entirely sure if I should also change this here, because you told me that you want to standardise tournament terminology anyway, but might be good to adjust it accordingly.